src/model/mod.rs

```diff
@@ -89,15 +89,17 @@ impl Deref for ContinuousAlleleFreqs {
 pub enum VariantType {
     Insertion(Option<Range<u32>>),
     Deletion(Option<Range<u32>>),
-    SNV
+    SNV,
+    Ref, // homozygous reference site
```

Comment:
Can we call this `None` instead?

Reply:
Jep, will refactor and double-check `pairwise.rs` later today. Will also add `TODO`s for looking at this again for (no-)indel calling -- or would you prefer this filed as an issue instead of in the code?

Reply:
I don't think we can do much in this regard. But yes, a TODO sounds reasonable.